container-runtime/src/Adapters/RpcListener.ts

```diff
@@ -342,3 +340,88 @@ export class RpcListener {
       })
   }
 }
+function getResult(
```

Comment:
Should be the tranformations that existed for the other two adapters, so all the error adaption should be here